includes/config.inc

```diff
@@ -531,6 +531,8 @@ function islandora_solr_metadata_config_field_form($form, &$form_state, $config_
     'min_wordsafe_length_input_path' => "wrapper[truncation][word_safe]",
   );
   islandora_solr_metadata_add_truncation_to_form($set, $truncation_config);
+  $permissions = $get_default(array('permissions'), array());
```

Comment:
On second thought, to avoid later `isset()` calls, instead of an empty array to have:
```php
$permissions = $get_default(array('permissions'), array(
  'enable_permissions' => FALSE,
  'permissions' => array(),
));
```
... otherwise, fields would throw warnings when first loaded, before they get a value.